packages/ckeditor5-table/src/tablecolumnresize/tableresizewidthcommand.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license Copyright (c) 2003-2022, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module
+ */
+
+import TableWidthCommand from '../tableproperties/commands/tablewidthcommand';
+
+/**
+ * @extends module:table/tableproperties/commands/tablewidthcommand~TableWidthCommand
+ */
+export default class TableResizeWidthCommand extends TableWidthCommand {
```

Comment:
I don't have it from the top of my head. I have a feeling that we covered it on one of integrating TC with column resize feature so you could technically find it on one of the recordings - though that will take a little while to find the proper one 🤔 You could check with @mmotyczynska first.

Reply:
I consulted the team and we came to the conclusion that at the beginning the implementation was different and it was reasonable to inherit from the `TableWidthCommand` instead of `TablePropertyCommand`, but later on as we simplified the code it became unnecessary, so I changed it back.

Reply:
Still we should explain why we are not using the other table width command and this is missing now.